examples/tests/valid/intrinsics.test.w

```diff
@@ -1,13 +1,39 @@
 bring fs;
 bring expect;
+bring cloud;
+bring util;
 bring "./subdir/bar.w" as bar;
 
+// @dirname
+
 let path = "SHOULD_IGNORE";
 let filename = "intrinsics.test.w";
 
 let currentFile = fs.join(@dirname, filename);
 expect.equal(filename, fs.basename(currentFile));
-
 expect.equal(@dirname, fs.dirname(currentFile));
-
 expect.equal(bar.Bar.getSubdir(), fs.join(@dirname, "subdir"));
+
+// @inflight
+
+let counter = new cloud.Counter();
+pub class Example {
+  pub inflight getMessage(): str {
+    return "message";
+  }
+  pub inflight done() {
+    counter.inc();
+  }
+}
+let example = new Example();
+let queue = new cloud.Queue();
+queue.setConsumer(@inflight("./inline_typescript.ts", 
+  lifts: { example: { lift: example, ops: ["getMessage", "done"] } }
```

Comment:
```suggestion
  lifts: { example: { obj: example, ops: ["getMessage", "done"] } }
```

This is so verbose...

Can we make this work?

```js
lifts: {
  example: ["getMessage", "done"],
  my_bucket: ["put"],
}
```

Since this is an intrinsic, I believe we can technically make this work, no? It does limit the user's ability to map the wing symbol to the context key, but I am not sure this is actually needed.

Reply:
I only did it this way because you mentioned wanting to map the symbol to a new context key, so I'm happy to change it to something more terse.
The bigger downside to me is we lose the ability to lift arbitrary preflight expressions (since it has to be a symbol). The workaround would be to use a `let` binding to get a symbol, which isn't horrible so I'll go with that.

Reply:
cool beans!

(i always reserve the right to change my mind)

Reply:
I made the syntax slightly less verbose but I struggled to implement the ideal one you proposed. A big problem is the DX for lifting primitives or not specifying ops didn't make sense because this isn't valid.

```wing
lifts: {
  example,
}
```

Reply:
I see (we discussed supporting such syntax in the future, like in JavaScript, right?).

So the new syntax is:

```js
lifts: {
  example: { obj: example }
}
```

Correct?

Reply:
No I wanted to still optimize the common case, so I went with an array instead:

```lifts
lifts: [
  { obj: example }
]
```

This will automatically use "example" as the alias. You can provide a different alias and ops in that object

Reply:
I see. OK. Not loving it, but let's get this moving.

Reply:
would you prefer

```wing
lifts: {
  example: { obj: example }
}
```

?

Reply:
That's basically the original proposal (with `lift` => `obj`). Correct?

Would it make sense to use @yoav-steinberg's syntax?

```js
lifts: [
  { b: [put, list] }, // <-- special syntax
]
```

Reply:
Also I was hoping to avoid giving intrinsics access to custom syntax, I'd much rather be able to compose existing language features in them

Reply:
Yes, I like the idea of coming up with a language syntax to describing a lift specification (e.g. `myBucket^[ put, list ]`) which resolves to an `std.Lift` expression and can be used anywhere that takes `std.Lift`. Then, `@inflight` just takes in `Array<std.Lift>`.

But I am realizing this is not for this PR.

So maybe it makes more sense not to create yet another special syntax and just use your original proposal (I prefer the term `obj` over `lift`).

Reply:
```
^myBucket.[put,list]
^myBucket.put
^myBucket.{put|list}
```

Reply:
What's wrong with the syntax currently used in this PR?

```wing
lifts: [
  { obj: example }
]
```

It's less verbose than the original and the syntax is not special. The way it's interpreted is special, but it's not causing any limitations for the implementation.

Reply:
I can't do this, right?

```js
lifts: [
  { obj: x.y.z.someClosure() }
]
```

Or even this:

```js
lifts: [
  { obj: myBucket.arn }
]
```

Feels like it's not fully baked...

Reply:
You can, you will just have to provide an alias (the compiler will give an error)

```wing
lifts: [
  { obj: myBucket.arn, alias: "bucketArn" }
]
```

I'll admit that I'm hoping to eventually replace this with something closer to what you were mentioning, but in the mean time this doesn't feel too bad